pkg/sync/v11/data/ClusterRole.rbac.authorization.k8s.io/privileged-creator.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: privileged-creator
```

Comment:
what's this for?

Reply:
Moved to ClusterRole customer-admin-cluster.